string_functions_upper_lower_swap.py

```diff
@@ -0,0 +1,15 @@
+# Input
+a = input("Enter your string: ")
+print (f"You entered :{a}")
+
+# Lower case
+b = str.lower(a)
```

Comment:
I know this works. But you should have used "b = a.lower()" because "a" is of type "str". Same with other methods in this program. Please change all of them